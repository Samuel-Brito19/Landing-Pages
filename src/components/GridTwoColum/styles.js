import styled, { css } from 'styled-components'

export const Container = styled.div`
${({ theme }) => css`
display: grid;
grid-template-columns: 1fr 2fr;
align-items: center;
gap: ${theme.spacings.large};
`}
`;

export const TextContainer = styled.div`
${({ theme }) => css``}
`;

export const ImageContainer = styled.div`
${({ theme }) => css``}
`;

export const Image = styled.img`
${({ theme }) => css``}
`;
