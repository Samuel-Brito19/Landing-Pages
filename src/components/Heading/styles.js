import { css, styled } from "styled-components";

export const Title = styled.h1`
${({ theme }) => css`
color: ${theme.colors.mainBg};

`};

`
