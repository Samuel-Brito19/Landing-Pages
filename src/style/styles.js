import styled, { css } from 'styled-components';


export const Wrap = styled.div`
  ${({ theme }) => css`
    background: ${theme.colors.mainBg};
  `}
`;
