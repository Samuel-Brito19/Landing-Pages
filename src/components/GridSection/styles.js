import styled, { css } from 'styled-components'
import { Title as HeadingContainer } from '../Heading/styles'

export const Container = styled.div`
${({ theme }) => css``}
`;

export const Grid = styled.div`
${({ theme }) => css`
counter-reset: grid-counter;
display: grid;
grid-template-columns: repeat(auto-fill, minMax(280px, 1fr));
gap: ${theme.spacings.large};
`}
`;

export const GridElement = styled.div`
${({ theme }) => css`
${HeadingContainer} {
  position: relative;
  left: 5rem;

}

${HeadingContainer}::before {
  counter-increment: grid-counter;
  content: counter(grid-counter);
  position: absolute;
  font-size: ${theme.font.sizes.xhuge};
  top: -2rem;
  left: -5rem;
  transform: rotate(5deg);


}
`}
`;
