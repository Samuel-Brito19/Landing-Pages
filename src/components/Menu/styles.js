import styled, { css } from 'styled-components'
import { Container as SectionContainer } from '../SectionContainer/styles';
import { Title as Heading } from '../Heading/styles';

export const Container = styled.div`
${({ theme }) => css`

position: fixed;
z-index: 5;
top: 0;
bottom: 0;
right: 0;
left: 0;
border-bottom: ${theme.colors.mediumGray};
background: ${theme.colors.white};

> ${SectionContainer} {
  display: grid;
  grid-template-columns: 1fr;
  grid-template-rows: 1fr;
  height: 100vh;
  align-items: center;
  overflow-y: auto;
}

& ${Heading} {
  padding-bottom: ${theme.spacings.large};
  display: flex;
  justify-content: center;
}

@media ${theme.media.lteMedium} {
  height: 100vh;
}

`}
`;

export const MenuContainer = styled.div`
${({ theme }) => css`
display: flex;
justify-content: space-between;
align-items: center;

@media ${theme.media.lteMedium} {
  display: block;
  text-align: center;
  padding: ${theme.spacings.xxlarge} 0;
}

`}
`

export const Button = styled.div`
${({ theme }) => css`
z-index: 6;
position: fixed;
top: 2rem;
right: 2rem;
width: 4rem;
height: 4rem;
background: ${theme.colors.primaryColor};
color: ${theme.colors.white};
border: none;

`}

@media ${theme.media.lteMedium} {
  display: flex;
  align-items: center;
  justify-content: center;
}


> svg {
  width: 2.5rem;
  height: 2.5rem
}
`
