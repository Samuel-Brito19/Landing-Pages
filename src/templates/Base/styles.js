import styled, { css } from 'styled-components'
import { Container as TextComponent } from '../../components/TextComponent/styles';

export const Container = styled.div`
${({ theme }) => css`
& ${TextComponent} {
  font-size: ${theme.font.sizes.small};
}


`}
`;
