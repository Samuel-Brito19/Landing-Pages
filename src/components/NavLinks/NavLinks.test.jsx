import { screen } from '@testing-library/react'
import { NavLinks } from '.'
import { renderTheme } from '../../templates/styles/renderTheme'
describe('<NavLinks/>', () => {
  it('should render', () => {
    renderTheme(<NavLinks>children</NavLinks>)
    expect(screen.getByRole('heading')).toBeInTheDocument()
  })
})
