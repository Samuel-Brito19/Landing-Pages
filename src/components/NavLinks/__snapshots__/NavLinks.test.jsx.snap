// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavLinks /> should match snapshot 1`] = `
[
  .c0 {
  display: block;
  text-decoration: none;
  font-size: 1.6rem;
  padding: 1.6rem;
  color: #0A1128;
  position: relative;
}

.c0::after {
  content: '';
  position: absolute;
  bottom: 0;
  left: 50%;
  width: 0;
  height: 0.2rem;
  background: #dc143c;
  transition: all 300ms ease-in-out;
}

.c0:hover::after {
  left: 25%;
  width: 50%;
}

@media (max-width: 768px) {

}

<a
    class="c0"
    href="#target1"
    target="_self"
  >
    <h1>
      Link 1
    </h1>
  </a>,
  .c0 {
  display: block;
  text-decoration: none;
  font-size: 1.6rem;
  padding: 1.6rem;
  color: #0A1128;
  position: relative;
}

.c0::after {
  content: '';
  position: absolute;
  bottom: 0;
  left: 50%;
  width: 0;
  height: 0.2rem;
  background: #dc143c;
  transition: all 300ms ease-in-out;
}

.c0:hover::after {
  left: 25%;
  width: 50%;
}

@media (max-width: 768px) {

}

<a
    class="c0"
    href="#target2"
    target="_self"
  >
    <h1>
      Link 2
    </h1>
  </a>,
  .c0 {
  display: block;
  text-decoration: none;
  font-size: 1.6rem;
  padding: 1.6rem;
  color: #0A1128;
  position: relative;
}

.c0::after {
  content: '';
  position: absolute;
  bottom: 0;
  left: 50%;
  width: 0;
  height: 0.2rem;
  background: #dc143c;
  transition: all 300ms ease-in-out;
}

.c0:hover::after {
  left: 25%;
  width: 50%;
}

@media (max-width: 768px) {

}

<a
    class="c0"
    href="#target3"
    target="_self"
  >
    <h1>
      Link 3
    </h1>
  </a>,
  .c0 {
  display: block;
  text-decoration: none;
  font-size: 1.6rem;
  padding: 1.6rem;
  color: #0A1128;
  position: relative;
}

.c0::after {
  content: '';
  position: absolute;
  bottom: 0;
  left: 50%;
  width: 0;
  height: 0.2rem;
  background: #dc143c;
  transition: all 300ms ease-in-out;
}

.c0:hover::after {
  left: 25%;
  width: 50%;
}

@media (max-width: 768px) {

}

<a
    class="c0"
    href="#target4"
    target="_self"
  >
    <h1>
      Link 4
    </h1>
  </a>,
  .c0 {
  display: block;
  text-decoration: none;
  font-size: 1.6rem;
  padding: 1.6rem;
  color: #0A1128;
  position: relative;
}

.c0::after {
  content: '';
  position: absolute;
  bottom: 0;
  left: 50%;
  width: 0;
  height: 0.2rem;
  background: #dc143c;
  transition: all 300ms ease-in-out;
}

.c0:hover::after {
  left: 25%;
  width: 50%;
}

@media (max-width: 768px) {

}

<a
    class="c0"
    href="#target5"
    target="_self"
  >
    <h1>
      Link 5
    </h1>
  </a>,
  .c0 {
  display: block;
  text-decoration: none;
  font-size: 1.6rem;
  padding: 1.6rem;
  color: #0A1128;
  position: relative;
}

.c0::after {
  content: '';
  position: absolute;
  bottom: 0;
  left: 50%;
  width: 0;
  height: 0.2rem;
  background: #dc143c;
  transition: all 300ms ease-in-out;
}

.c0:hover::after {
  left: 25%;
  width: 50%;
}

@media (max-width: 768px) {

}

<a
    class="c0"
    href="#target6"
    target="_self"
  >
    <h1>
      Link 6
    </h1>
  </a>,
  .c0 {
  display: block;
  text-decoration: none;
  font-size: 1.6rem;
  padding: 1.6rem;
  color: #0A1128;
  position: relative;
}

.c0::after {
  content: '';
  position: absolute;
  bottom: 0;
  left: 50%;
  width: 0;
  height: 0.2rem;
  background: #dc143c;
  transition: all 300ms ease-in-out;
}

.c0:hover::after {
  left: 25%;
  width: 50%;
}

@media (max-width: 768px) {

}

<a
    class="c0"
    href="#target7"
    target="_self"
  >
    <h1>
      Link 7
    </h1>
  </a>,
  .c0 {
  display: block;
  text-decoration: none;
  font-size: 1.6rem;
  padding: 1.6rem;
  color: #0A1128;
  position: relative;
}

.c0::after {
  content: '';
  position: absolute;
  bottom: 0;
  left: 50%;
  width: 0;
  height: 0.2rem;
  background: #dc143c;
  transition: all 300ms ease-in-out;
}

.c0:hover::after {
  left: 25%;
  width: 50%;
}

@media (max-width: 768px) {

}

<a
    class="c0"
    href="#target8"
    target="_self"
  >
    <h1>
      Link 8
    </h1>
  </a>,
  .c0 {
  display: block;
  text-decoration: none;
  font-size: 1.6rem;
  padding: 1.6rem;
  color: #0A1128;
  position: relative;
}

.c0::after {
  content: '';
  position: absolute;
  bottom: 0;
  left: 50%;
  width: 0;
  height: 0.2rem;
  background: #dc143c;
  transition: all 300ms ease-in-out;
}

.c0:hover::after {
  left: 25%;
  width: 50%;
}

@media (max-width: 768px) {

}

<a
    class="c0"
    href="#target9"
    target="_self"
  >
    <h1>
      Link 9
    </h1>
  </a>,
  .c0 {
  display: block;
  text-decoration: none;
  font-size: 1.6rem;
  padding: 1.6rem;
  color: #0A1128;
  position: relative;
}

.c0::after {
  content: '';
  position: absolute;
  bottom: 0;
  left: 50%;
  width: 0;
  height: 0.2rem;
  background: #dc143c;
  transition: all 300ms ease-in-out;
}

.c0:hover::after {
  left: 25%;
  width: 50%;
}

@media (max-width: 768px) {

}

<a
    class="c0"
    href="#target10"
    target="_self"
  >
    <h1>
      Link 10
    </h1>
  </a>,
]
`;
